doc/contributing/investigating-native-memory-leaks.md

```diff
@@ -1,28 +1,28 @@
 # Investigating memory leaks with valgrind
 
-A Node.js process may run out of memory due to excessive consumption of
-native memory. Native Memory is memory which is not managed by the
-V8 Garbage collector and is allocated either by the Node.js runtime, its
-dependencies or native [addons](https://nodejs.org/docs/latest/api/n-api.html).
+ A Node.js process may run out of memory due to excessive consumption of
+ native memory. Native Memory is memory which is not managed by the
+ V8 Garbage collector and is allocated either by the Node.js runtime, its
+ dependencies or native [addons](https://nodejs.org/docs/latest/api/n-api.html).
```

Comment:
Why those spaces at the beginning of the line?

Reply:
I didn't even know they existed; whatever they are, I wasn't the one to write them in... probably VS Code trying to sabotage me

Reply:
Should be fixed now.